src/core.js

```diff
@@ -262,7 +262,7 @@ jQuery.extend( {
 			return obj + "";
 		}
 
-		// Support: Android<4.0 (functionish RegExp)
+		// Support: Android 2.3 only (functionish RegExp)
```

Comment:
I think all lines like these, including "IE 9 only" and "IE 9 - 11 only", should include the initial `<=`. Unless, of course, they are confirmed to _not_ be relevant in e.g. Android 2.2 or IE 8. I wouldn't ask except that it's part of the reason for this PR.